pyclesperanto_prototype/_tier0/_pycl.py

```diff
@@ -99,43 +99,50 @@ def assert_bufs_type(mytype, *bufs):
             % (mytype, str([b.dtype.type for b in bufs]))
         )
 
+def prepare(arr):
+    return np.require(arr, None, "C")
 
-def _wrap_OCLArray(cls):
-    """
-    WRAPPER
-    """
+class OCLArray(array.Array, np.lib.mixins.NDArrayOperatorsMixin):
```

Comment:
you can probably remove the vendored from GPUtools and license, since this now bears little similarity.